src/components/course-page-tags.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tag from './tags.js';
+
+// For any questions please refer to Timmy
```

Comment:
??? gitlens will blame the author of code already, no need to write this and pollute code base